### DISCUSSION TITLE
WinUI Galleries - Store Update 🎉💻

### DISCUSSION THREAD

**Discussion by gregwoo-microsoft** [microsoft/WinUI-Gallery#939] [Announcements]
Hello! 👋 We are excited to announce that the WinUI Gallery apps will soon be updated to the Microsoft Store. Those updates will include many changes that a lot of you have been waiting for:

1. 	**New app names**

- The XAML Controls Gallery and the WinUI 3 Controls Gallery are getting new names! They will be called the WinUI 2 Gallery and the WinUI 3 Gallery respectively. This change creates a consistent and simplified naming scheme. It also helps better understand which UI library each app uses.
- You may also have noticed that the repo has also been renamed "WinUI-Gallery" (previously “XAML Controls Gallery”) to reflect this change.

2. **Visual revamp of both apps**
- A new home page has been added to both apps with a refreshed set of control icons. We have also added tiles in the home page to help you access frequently used web pages. Let us know if you think that new tiles should be added!

3. **Structural changes**
- The "main" branch will now host the WinUI 3 Gallery app code and a separate "winui2" branch will host the code for the WinUI 2 Gallery. We hope that by sharing this, we can prevent any conflicts with local copies of this repo that you may have.
- The folder structure of the repo will also be slightly altered to be more consistent across apps (for instance, the "XAMLControlsGallery" project folder will be renamed to "WinUI-Gallery" and a new asset folder called "ControlImages" folder has been added to contain all of the new icons).

4. **New controls, samples, and bug fixes**
- We have added new controls such as the "System Backdrops Mica/Acrylic" and "Create Multiple Windows" pages. Several control pages have also been fixed or updated such as the "ProgressRing" or "FlipView" pages.
 
We hope that you are as thrilled as we are about these changes! Feel free to let us know if you have any feedback or questions.

**Comment by bpulliam** [comment]
![Announce](https://user-images.githubusercontent.com/7055169/168142642-0e36ab5b-2a22-401c-ad6c-d8b4adf4828f.jpg)

**Reply by renzet** [reply]
This screenshot shows WinAppSDK 1.1, but I downloaded the app from the store (version 1.3.12.0) and it shows WinAppSDk 1.0. How can I get the 1.1 version? I tried running the app from the Main branch, but got the error MSB3774: Could not find SDK "Microsoft.UniversalCRT.Debug, Version=10.0.18362.0" in Visual Studio 2022.

**Reply by gregwoo-microsoft** [reply]
The Store version is using WASDK 1.0 as the 1.1 version isn't released yet. In the main branch of the repo, we are using WASDK 1.1 preview 3. Regarding your error, I suggest that you check out this issue: https://github.com/microsoft/WinUI-Gallery/issues/871 

I hope that this helps! :)

**Reply by renzet** [reply]
Thanks, the steps outlined in #871 fixed the build error and I was able to run the version from the `main` branch

**Comment by gregwoo-microsoft** [comment]
The public repo has been updated:
-	winui3 -> main
-	master -> winui2

On your local repo, you can do:
-	Git fetch
-	Git branch -d master
-	Git branch -d winui3
-	Git checkout main
-	Git checkout winui2

**Reply by bpulliam** [reply]
The store has been updated! https://www.microsoft.com/store/productId/9P3JFPWWDZRC

**Reply by FinnurHrafn** [reply]
The app store still gives me the old version where I live in Iceland.

**Reply by gregwoo-microsoft** [reply]
The Store version is using WASDK 1.0 as the 1.1 version isn't released yet. We plan to release an update as soon as it is available. You can use the main branch of this repo in the meantime (it is using WASDK 1.1 preview 3). I hope that this helps :)
